java/src/main/java/ai/onnxruntime/OnnxTensor.java

```diff
@@ -108,11 +87,6 @@ public Object getValue() throws OrtException {
     }
   }
 
-  @Override
-  public TensorInfo getInfo() {
-    return info;
-  }
-
   @Override
```

Comment:
>@Override [](http://example.com/codeflow?start=2&length=9)

Is this a public interface?

Reply:
I'm unsure what you mean. This comment is attached to the override annotation on the `toString` method, which `OnnxTensor` inherits from `Object` like all Java objects.

Reply:
What I meant is getInfo() used to be a public interface. Is it Ok to remove()

Reply:
Ah, I moved `getInfo` up to `OnnxTensorLike` as it's shared between the dense and sparse tensors.